src/platform.deno.ts

```diff
@@ -87,20 +84,29 @@ export class InputFile {
         }
         this.filename = filename;
     }
-    get [inputFileData]() {
+    async [toRaw](): Promise<Uint8Array | AsyncIterable<Uint8Array>> {
```

Comment:
Could be simplified to
```suggestion
    async [toRaw](): AsyncIterable<Uint8Array> {
```

Reply:
No, since we're using `await`.

Reply:
grammY 2 should ideally achieve web `File` compat:

```ts
stream(): ReadableStream<Uint8Array> {
```

Reply:
I'm not sure about this, but either way this is irrelevant now.